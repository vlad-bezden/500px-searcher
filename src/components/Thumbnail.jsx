import React from 'react'

const Thumbnail = ({ src }) => (
    <div>
        <img src={src} />
    </div>
)

Thumbnail.propTypes = {
  src: React.PropTypes.string.isRequired
}

export default Thumbnail