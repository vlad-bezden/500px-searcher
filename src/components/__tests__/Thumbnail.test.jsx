import React from 'react'
import { shallow } from 'enzyme'

import Thumbnail from '../Thumbnail'

test('Thumbnail renders correctly', () => {
  const component = shallow(
    <Thumbnail src='http://test.com/test.jpg' />
  )
  const tree = component.getNode()

  expect(tree).toMatchSnapshot()
})