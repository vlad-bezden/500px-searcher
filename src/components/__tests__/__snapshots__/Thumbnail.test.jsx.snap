// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thumbnail renders correctly 1`] = `
<div>
  <img
    src="http://test.com/test.jpg"
  />
</div>
`;
